### PR TITLE
Test against Django 1.7

### PR DESCRIPTION
**Pull request by pipermerriam**
### What is the problem / feature ?

It would be nice to know that `django-widgy` will work with Django 1.7

### How did it get fixed / implemented ?

- Added tests for 1.7 to the `travis.yml` matrix.
- Added call to `django.setup()` in `runtests.py` to load the app registry.

### How can someone test / see it ?

Take a look at travis or run the tests locally.

*Here is a cute animal picture for your troubles...*

baby turtle perches on top of succulent strawberry twice his size on fruit farm 1

### CODE REVIEW

**Comment by pipermerriam** [comment]
This is currently a work in progress.  Just opening the PR early.

**Comment by rockymeza** [comment]
Could you please rebase on the new master?

**Comment by gavinwahl** [comment]
Yeah, this has a bunch of conflicts

**Comment by gavinwahl** [comment]
I think #303 is required for this to work as well

File: widgy/compat.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import django
+
+
+def check_if_field_name_exists_on_cls(field_name, cls):
+    """
+    Check if a field is already present on a model in a `safe` manner that does
+    not trigger or violate application loading.
+    """
+    if django.get_version() < "1.7":
```
Comment (gavinwahl) [inline widgy/compat.py]: Is this a real thing? I've always done `django.VERSION < (1, 7)`

Comment (pipermerriam) [inline widgy/compat.py]:
```python
In [1]: '1.6' < '1.7'
Out[1]: True

In [2]: '1.7' < '1.6'
Out[2]: False

In [3]: '1.6.4' < '1.7'
Out[3]: True

In [4]: '1.7' < '1.6.4'
Out[4]: False
```
Seems to.

Comment (acatton) [inline widgy/compat.py]: @pipermerriam this is string comparison. It doesn't work at all for numbers:
```
>>> '10.2' < '10.15'
False
```

Comment (pipermerriam) [inline widgy/compat.py]: Cool, I was sort of remembering that there was a case where this didn't work but I couldn't figure out what it was.

Comment (acatton) [inline widgy/compat.py]: @pipermerriam I edited my answer for clarification. You should be able to figure out the case it doesn't work next time ;) .

**Comment by pipermerriam** [comment]
Closing in favor of #303

**Pull request closed without merging**